Makefile

```diff
@@ -36,13 +36,9 @@ doc_reqs:
 
 api-docs: FLAGS := $(if $(FLAGS),$(FLAGS),--config=config.yaml)
 api-docs: | doc_reqs
-	@PYTHONPATH=. python tools/openapi/build-spec.py $(FLAGS)
-	npx @redocly/cli build-docs openapi.json \
-          --title "SkyPortal API docs" \
-          --theme.openapi.logo.gutter 2rem
+	@PYTHONPATH=. python tools/docs/build-spec.py $(FLAGS)
+	@PYTHONPATH=. python tools/docs/patch-template.py $(FLAGS)
```

Comment:
Perhaps a more descriptive name? Something like `patch-api-doc-template.py`. And we could rename the other one to `build-api-spec.py`.

Reply:
done